setup.py

```diff
@@ -85,7 +85,7 @@
     install_requires=[
         "dask",
         "diffsims       >= 0.5",
-        "hyperspy       >= 1.7.0",  # significant improvements
+        "hyperspy       >= 1.7.0, <2.0.0",  # significant improvements
```

Comment:
```suggestion
        "hyperspy       >= 1.7.0, <2.0ra0",  # significant improvements
```

As "2.0rc0" could be installed if the `--pre` flag is used:

```python
>>> from packaging.version import Version
>>> print(Version("2.0rc0") < Version("2.0.0"))
True
>>> print(Version("2.0rc0") < Version("2.0a0"))
False
```